en/02_Developer_Guides/00_Model/13_Managing_Records.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Managing Records
+summary: Manage your DataObject records
+icon: list-alt
+---
+
+# Managing Records
+
+Most records in Silverstripe CMS are managed [in a GridField](../forms/field_types/gridfield) - whether in the [GridField](api:SilverStripe\Forms\GridField\GridField)
+of some other record or directly [in a ModelAdmin](../customising_the_admin_interface/modeladmin/). The notable exceptions to this are
+[SiteConfig](api:SilverStripe\SiteConfig\SiteConfig) and [SiteTree](api:SilverStripe\CMS\Model\SiteTree).
+
+## Getting an edit link
+
+To get a link for editing a record, other than some exceptions like `SiteTree`, you have historically had to generate the link mostly
```

Comment:
I don't think we need to provide a historical background. The docs we provide are always for the latest version. Unfortunately we don't contain docs for older minor versions, though that's just a limitation of our documentation system, and not something we should attempt to handle here